tallycash/extension
# Chain Specific dapp Permissions

Closes #1581 

Refactors `dapp-permissions` redux store and the `provider-bridge` db to have network-aware `PermissionRequests`.

### To Test
-  checkout `main` 
-  import a wallet
- grant permission to uniswap
- checkout `chain-specific-permissions`
- update extension
- [ ] you should see a permission in uniswap with `chainID = 1` in the provider-bridge IndexDB.
- [ ] you should see a permission in uniswap with `chainID = 137` in the provider-bridge IndexDB.
- [ ] granting permissions to new dapps should work as before - and it should grant permission to all networks.
- [ ] revoking permission from a dapp should revoke permission on all networks (can be verified through indexDB).
